frontend/src/metabase/dashboard/components/PublicLinkPopover/DashboardPublicLinkPopover.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Dashboard } from "metabase-types/api";
```

Comment:
I split the popover into `Dashboard` and `Question` popovers to separate the different actions and the fact that getting a `uuid` for questions and dashboards is completely different.